backendImpl/src/main/java/com/virtuslab/gitmachete/backend/impl/GitMacheteRepository.java

```diff
@@ -774,4 +781,41 @@ IGitMacheteRepositorySnapshot discoverGitMacheteRepository() throws GitMacheteEx
     }
 
   }
+
+  @Getter
+  private static class GitMacheteNonRootBranchCreationResult {
```

Comment:
Although I don't like double plural like _busses stops_, I'd rather use `GitMacheteNonRootBranchesCreationResult` for clarity